src/appleseed/renderer/kernel/lighting/pt/ptlightingengine.cpp

```diff
@@ -748,6 +800,36 @@ Dictionary PTLightingEngineFactory::get_params_metadata()
             .insert("help", "Maximum number of path bounces"));
 
     metadata.dictionaries().insert(
+        "specular_max_path_length",
+        Dictionary()
+            .insert("type", "int")
+            .insert("default", "8")
+            .insert("unlimited", "true")
+            .insert("min", "1")
```

Comment:
The minimum number of bounces should be 0, not 1.

Bounces and path lengths are different things: a path of length 1 has 0 bounce. A path with one bounce has length 2. Etc.